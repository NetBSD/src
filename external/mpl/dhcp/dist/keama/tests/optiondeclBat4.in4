# bad Bat (non terminal array) option declaration

option space foobar;
option foobar.fmt-Bat code 1 = { array of unsigned integer 8, text };


