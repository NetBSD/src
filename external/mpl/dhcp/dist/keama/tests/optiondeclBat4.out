{
  # bad Bat (non terminal array) option declaration
  "Dhcp4": {
    "option-def": [
      /// unsupported array inside a record
      {
        "space": "foobar",
        "name": "fmt-Bat",
        "code": 1,
//      "array": true,
//      "definition": "{ array of uint8, string}",
        /// Option definition is not compatible with Kea
        /// Fallback to full binary
        "type": "binary"
      }
    ]
  }
}
